.github/workflows/test.yaml

```diff
@@ -73,6 +72,7 @@ jobs:
           [[ $description == "empty image" ]]
       - name: Delete test cache.
         if: always()
+        # checkov:skip=CKV_GHA_3: Suppress error "Suspicious use of curl with secrets" for Delete test cache step.
```

Comment:
Moot: Omit "for Delete test cache step" from the comment since this is hopefully self-evident from the placement of the comment.